app/src/main/java/inspire2connect/inspire2connect/home/homeActivity.java

```diff
@@ -211,17 +211,18 @@ public void onClick(View view) {
             }
         }
 
+//        if (view == ll_but[0]) {
+//            i = getGovernmentIntent(this);
+//        } else if (view == ll_but[1]) {
+//            i = new Intent(homeActivity.this, QuestionsActivity.class);
+//        } else
         if (view == ll_but[0]) {
-            i = getGovernmentIntent(this);
-        } else if (view == ll_but[1]) {
-            i = new Intent(homeActivity.this, QuestionsActivity.class);
-        } else if (view == ll_but[2]) {
             i = new Intent(homeActivity.this, ContactTracerActivity.class);
-        } else  if (view == ll_but[3]) {
+        } else  if (view == ll_but[1]) {
             i = new Intent(homeActivity.this, onAIrActivity.class);
-        }else if (view == ll_but[4]) {
+        }else if (view == ll_but[2]) {
             i = new Intent(homeActivity.this, selectChatBotActivity.class);
```

Comment:
direct the option to satya directly instead of opening a window to select chatbot.

Reply:
Done